what.html

```diff
@@ -7,41 +7,110 @@
+                    </ul>
+                    <h4>Business</h4>
+                    <ul class="biz">
+                        <li>Brockerage Agreement</li><br>
+                        <li>Business Formation</li><br>
+                        <li>Business Related Contracts</li><br>
+                        <li>Labor/Employment</li>
+                    </ul>
+                    <h4>Transportation</h4>
+                    <p class="aligncenter">
+                    <img src="https://images.pexels.com/photos/225502/pexels-photo-225502.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940" id="redbox_pic" width="150" height="84"></p>
+                    <b><h4>Attorneys</h4></b>
+                    <h5>Meet our prestigious partners and esteemed associates.</h5>  
+                </div>
+        
+                </div>
+                
+                <!-- Rectangle Images Terrell-->
+                <!-- <div class="rectangles" >
+                    <div class="grid-wrapper">
+                        <div class="bx1"><img src="https://via.placeholder.com/200x100" alt=""></div>
+                        <div class="bx2"><img src="https://via.placeholder.com/200x100" alt=""></div>
+                        <div class="bx3"><img src="https://via.placeholder.com/200x100" alt=""></div>
+                        <div class="bx4"><img src="https://via.placeholder.com/200x100" alt=""></div>
+                        <div class="bx5"><img src="https://via.placeholder.com/200x100" alt=""></div>
+                        <div class="bx6"><img src="https://via.placeholder.com/200x100" alt=""></div>
+                    </div>
+                </div> -->
+        
+                <!-- Footer Josey -->
+                <div class="footer">
+                    <hr class=hr3>
+                    <div class="navbar">
+                        <ul id= bottomnav>
+                            <li><a href="/who.html">WHO WE ARE |</a></li>
+                            <li><a href="/what.html">WHAT WE DO |</a></li>
+                            <li><a href="https://nzalaw.com/news/">NEWS & EVENTS |</a></li>
+                            <li><a href="https://nzalaw.com/practice-group/commercial/">WHERE WE WORK |</a></li>
+                            <li><a href="https://nzalaw.com/contact/">CONTACT US</a></li>
+                        </ul>
+                    <button><a href="/who.html">   WHO WE ARE |</a></button>
+                    <button><a href="/what.html"> WHAT WE DO |</a></button>
+                    <button><a href="https://nzalaw.com/news/"> NEWS & EVENTS |</a></button>
+                    <button><a href="https://nzalaw.com/practice-group/commercial/"> WHERE WE WORK |</a></button>
+                    <button><a href="https://nzalaw.com/contact/">CONTACT US</a></button>
+                        
+                    <p class="copyright"> ©2020 Chicago Coding Temple // Website Design by the Terlan Repository</p>
+        
+                    </div>
```

Comment:
Hey, I'm still learning how to use git but I see these changes from that footer earlier.  I'm definitely adding this part into the doc. Thanks!!